### ISSUE TITLE
Certificates not shown when issued by cert-manager

### ISSUE THREAD

**Issue by msalles-winn** [rancher/rancher#14423]
**Rancher versions:**
rancher/server or rancher/rancher: 2.0.2
rancher/agent or rancher/rancher-agent: 2.0.2

**Infrastructure Stack versions:**
kubernetes (if applicable): 1.8

**Docker version: (`docker version`,`docker info` preferred)**
Containers: 77
 Running: 41
 Paused: 0
 Stopped: 36
Images: 34
Server Version: 1.13.1
Storage Driver: aufs
 Root Dir: /home/docker/aufs
 Backing Filesystem: extfs
 Dirs: 368
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: aa8187dbd3b7ad67d8e5e3a15115d3eef43a7ed1
runc version: 9df8b306d01f59d3a8029be411de015b7304dd8f
init version: N/A (expected: 949e6facb77383876aeff8a6944dde66b3089574)
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 4.13.0-43-generic
Operating System: Ubuntu 17.10
OSType: linux
Architecture: x86_64
CPUs: 8
Total Memory: 62.88 GiB
Name: ns3116625
ID: 6JUF:ZNHI:6AAA:IO3T:EZKE:MY35:BWRA:57K5:C5NY:W7XN:QGL4:JTCN
Docker Root Dir: /home/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

**Operating system and kernel: (`cat /etc/os-release`, `uname -r` preferred)**
4.13.0-43-generic

**Type/provider of hosts: (VirtualBox/Bare-metal/AWS/GCE/DO)**
Bare metal

**Setup details: (single node rancher vs. HA rancher, internal DB vs. external DB)**
single node rancher

**Environment Template: (Cattle/Kubernetes/Swarm/Mesos)**
Kubernetes

**Steps to Reproduce:**
Create a cluster ;
Add a node
Install gitlab-ce image
create an ingress to serve gitlab on 80 for subdomain
Enable Helm
Install cert-manager
In console, change the .kube/config with the cluster configuration

** Create issuer : **
apiVersion: certmanager.k8s.io/v1alpha1
kind: Issuer
metadata:
  name: letsencrypt-issuer-prod
spec:
  acme:
    server: https://acme-v02.api.letsencrypt.org/directory
    email: <email>
    privateKeySecretRef:
      name: letsencrypt-issuer-prod
    http01: {}

** Result of the issuer : **
Status:
  Acme:
    Uri:  https://acme-v02.api.letsencrypt.org/acme/acct/<xxxx>
  Conditions:
    Last Transition Time:  ...
    Message:               The ACME account was registered with the ACME server
    Reason:                ACMEAccountRegistered
    Status:                True
    Type:                  Ready
Events:                    <none>

**Create certificate :** 
apiVersion: certmanager.k8s.io/v1alpha1
kind: Certificate
metadata:
  name: xxx
spec:
  secretName: xxx-tls
  dnsNames:
  - dom1
  - dom2
  - dom3
  issuerRef:
    name: letsencrypt-prod
    kind: ClusterIssuer
  acme:
      config:
      - http01:
          ingressClass: nginx
        domains:
        - dom1
        - dom2
        - dom3

** Result of the certificate : **

Events:
  Type    Reason          Age   From          Message
  ----    ------          ----  ----          -------
  Normal  CreateOrder     22m   cert-manager  Created new ACME order, attempting validation...
  Normal  DomainVerified  20m   cert-manager  Domain "dom" verified with "http-01" validation
  Normal  IssueCert       20m   cert-manager  Issuing certificate...
  Normal  CertObtained    20m   cert-manager  Obtained certificate from ACME server
  Normal  CertIssued      20m   cert-manager  Certificate issued successfully

**Results:**
The certificate page is still empty.

**Comment by ekristen** [comment]
I'm seeing the certificates showing in 2.0.6, but they aren't displayed correctly and deleting them doesn't seem to delete them from kubernetes either.  I've also found that cert-manager doesn't work out of the box on rancher without some manual editing of ingress `spec` settings. See https://github.com/rancher/rancher/issues/14924

**Comment by zebpalmer** [comment]
@msalles-winn not sure if you ever got this working... but for anyone that finds this: 

I had a brand new cluster on which rancher UI was not showing certificates issued by cert-manager even though I had that cluster configured "exactly" like one that was. I eventually discovered that the only difference was cert-manager was installed into the default namespace on the working clusters I went to recreate it in the default workspace at which point I noticed that the default workspace wasn't an option. At some point (I have no idea how) the default namespace was removed from the default project. Added it back and now I have all my cert-manager issued certs showing up and available for ingress assignment.  

  _fwiw I'm using a clusterissuer and wildcard domains -- setup similar to this post https://www.idealcoders.com/posts/rancher/2018/06/rancher-2-x-and-lets-encrypt-with-cert-manager-and-nginx-ingress/_
